source/content/guides/autopilot/01-introduction.md

```diff
@@ -17,6 +17,7 @@ audience: [development]
 product: [autopilot]
 integration: [--]
 ---
+<Wistia src="g6syd1p6ed" />
```

Comment:
@rachelwhitton should videos with content that is essentially equivalent to the text of the entire page it is on be right below the H1 as it is here, or below the first paragraph, like in `source/content/add-site-dashboard.md`